xml/System/String.xml

```diff
@@ -5410,15 +5410,15 @@ Invalid sequences are represented in the enumeration by <xref:System.Text.Rune.R
 
  A format item has the following elements:
 
- *index*
+ *index*\
```

Comment:
Did you want italic here, or bold? (3 occurrences)

Reply:
I'm not sure, but since I didn't introduce it, I'm going to leave it as-is for now. There are a lot of improvements we can make to this file, like moving out large swaths of text into conceptual docs.